dazel/lib/src/bazelify/build.dart

```diff
@@ -287,18 +299,38 @@ class DartLibrary implements DartBuildRule {
 
   @override
   String toRule(Map<String, BazelifyConfig> bazelifyConfigs) {
-    if (builders != null || generateFor != null) {
-      throw new UnimplementedError('`builders` and `generate_for` are not yet '
-          'supported for `target` configs.');
+    var rule = new StringBuffer();
+    var generatedTargets = <String>[];
+    if (builders != null) {
+      for (var builderName in builders.keys) {
+        var targetName = '${name}_$builderName';
+        generatedTargets.add(targetName);
+        var generateForGlob = generateFor == null
+            ? ''
+            : '    generate_for = ${_sourcesToGlob(generateFor, const [])},';
+        rule
+          ..writeln('$builderName(')
+          ..writeln('    name = "$targetName",')
+          ..writeln('    srcs = ${_sourcesToGlob(sources, excludeSources)},')
+          ..writeln(generateForGlob)
+          ..writeln(')');
+      }
     }
-    return '# Generated automatically for package:$package\n'
-        'dart_library(\n'
-        '    name = "$name",\n'
-        '    srcs = ${_sourcesToGlob(sources, excludeSources)},\n'
-        '    deps = ${depsToBazelTargetsString(dependencies, bazelifyConfigs)},\n'
-        '    enable_ddc = ${enableDdc ? 1 : 0},\n'
-        '    pub_pkg_name = "$package",\n'
-        ')';
+    var generatedSrcs = generatedTargets.isEmpty
+        ? ''
+        : ' + [${generatedTargets.map((t) => '":$t"').join(', ')}]';
+    rule
+      ..writeln('# Generated automatically for package:$package')
```

Comment:
Made things line up. We're inconsistent on when we use `writeln` and a buffer, at some point we might want to refactor and try to pick one style and stick to it.